src/features/HelpAndKnowledge/SolutionsHelp/SolutionDetails/Solutions/Generic/_components/TeamMemberForm/index.tsx

```diff
@@ -30,6 +30,7 @@ import { TeamMemberModeType } from '../MailboxAndTeamMemberModal';
 
 type FormValues = {
   userName: string;
+  displayName: string;
   role: string;
   isPrimary: boolean;
   receiveEmails: boolean;
```

Comment:
Again sorry I overlooked this earlier, but I think we want to tie to form values to the generated query type, rather than redefine the type